src/NzbDrone.Core.Test/NzbDrone.Core.Test.csproj

```diff
@@ -348,6 +349,10 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="..\..\Logo\1024.png">
+      <Link>Files\1024.png</Link>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
lets do if modified, should do that for other files as well, no point in always copying files during build